packages/itwinui-react/src/core/Menu/MenuItem.test.tsx

```diff
@@ -288,3 +287,52 @@ it('should handle key press with sub menus', async () => {
   await userEvent.keyboard('{Spacebar}');
   expect(mockedSubOnClick).toHaveBeenNthCalledWith(3, 'test_value_sub');
 });
+
+it('should have only one active submenu at a time', async () => {
+  render(
+    <MenuItem
+      key={0}
+      data-testid='parent'
+      subMenuItems={[
+        <MenuItem
+          key={1}
+          data-testid='sub-1'
+          subMenuItems={[
+            <MenuItem key={2} data-testid='sub-2'>
+              Test sub sub
+            </MenuItem>,
+          ]}
+        >
+          Test sub
+        </MenuItem>,
+        <MenuItem
+          key={3}
+          data-testid='sub-3'
+          subMenuItems={[
+            <MenuItem key={4} data-testid='sub-4'>
+              Test sub sub
+            </MenuItem>,
+          ]}
+        >
+          Test sub
+        </MenuItem>,
+      ]}
+    >
+      Test item
+    </MenuItem>,
+  );
+
+  const menuItem = screen.getByTestId('parent');
+
+  // focus to open sub menu
+  act(() => menuItem.focus());
+
+  // go right to move focus to the first submenu item. The first submenu's submenu should open.
```

Comment:
Similar to the above comment. I don't know if the wording is intentional but it feels a bit clunky.

Reply:
We could try combining the two sentences:

```suggestion
  // go right to move focus to the first submenu item and open its submenu.
```

… but I'm not sure if it makes it clear that the submenu's submenu should open. i.e. what are we testing for.

Or we could split it on two lines.

```diff
- // go right to move focus to the first submenu item. The first submenu's submenu should open.
+ // Go right to move focus to the first submenu item.
+ // The first submenu's submenu should open.
await userEvent.keyboard('{ArrowRight}');
expect(screen.getByTestId('sub-1')).toHaveFocus();
expect(screen.getByTestId('sub-2')).toBeVisible();

- // hover over the second submenu item. The first submenu's submenu should close.
+ // Hover over the second submenu item.
+ // The first submenu's submenu should close.
fireEvent.mouseEnter(screen.getByTestId('sub-3'));
expect(screen.queryByTestId('sub-4')).toBeFalsy();
```

Or maybe do you have any other suggestions? Because I'm not quite sure I understood what you meant by "clunky".

Reply:
I think the first option is good! I guess more specially the grammar seemed a bit off. But the first option is great

Reply:
Great, combined the two sentences similar to the first option (https://github.com/iTwin/iTwinUI/pull/1942/commits/c507a95f33585f9c25a646d8b4f0ff42c82e9fbb).